ui/src/components/admin/Notifications/PublishProgress.js

```diff
@@ -0,0 +1,289 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useContext, useEffect, useRef, useState } from 'react';
+
+import {
+  acknowledgeBulkPublishStatus,
+  stopAllPublishes,
+} from 'components/admin/Model/actions/Publish';
+import usePublishNotifications from 'components/admin/Notifications/PublishNotifications';
+import { ModelManagerContext } from 'components/admin/ModelsManager/ModelManagerController';
+import useConfirmationModal from 'components/modals/ConfirmationModal';
+import { NotificationsContext } from './NotificationsController';
+import NOTIFICATION_TYPES from './NotificationTypes';
+
+import CheckmarkIcon from 'icons/CheckmarkIcon';
+import CrossCircleIcon from 'icons/CrossCircleIcon';
+import CrossIcon from 'icons/CrossIcon';
+
+import { ButtonPill } from 'theme/adminControlsStyles';
+import {
+  Notification,
+  Message,
+  Details,
+  closeIcon,
+  closeIconDisabled,
+  ProgressBarContainer,
+  ProgressBarWrapper,
+  ProgressBarSectionComplete,
+  ProgressBarSectionFailed,
+  ProgressBarSectionIncomplete,
+  ProgressBarLabel,
+} from 'theme/adminNotificationStyles';
+import { Row, Col } from 'theme/system';
+import base from 'theme';
+
+import { PUBLISH_STATUS, PUBLISH_TYPES } from 'utils/constants';
+
+const {
+  keyedPalette: { trout },
+} = base;
+
+const BulkPublishState = {
+  complete: 'COMPLETE',
+  stopped: 'STOPPED',
+  publishing: 'PUBLISHING',
+  off: 'OFF',
+};
+
+const PublishProgress = ({ renderIcon }) => {
```

Comment:
Progress banner for publish progress. Very similar code to the import [ProgressBanner](https://github.com/nci-hcmi-catalog/portal/blob/develop/ui/src/components/admin/Notifications/ProgressBanner.js).

Pretty much the same thing as the aforementioned ProgressBanner, but publish-specific.

Definitely a future ticket/refactor where we make a generic progress banner component which takes in the status, notifications, labels, etc. as props. But again, time constraints 😓